teachertool/src/components/SplitPane.tsx

```diff
@@ -9,18 +9,20 @@ interface IProps {
     defaultSize: number | string;
     primary: "left" | "right";
     children: React.ReactNode;
+    leftPaneClass?: string;
+    rightPaneClass?: string;
 }
 
-export const SplitPane: React.FC<IProps> = ({ className, split, children }) => {
+export const SplitPane: React.FC<IProps> = ({ className, split, children, leftPaneClass, rightPaneClass }) => {
```

Comment:
The purpose is to enable the parent to inject css into the div that wraps the panel content. I may end up removing it. This can be accomplished without adding a prop.

Reply:
Removed!